test/InstallationScript.Tests/GivenThatIWantToInstallTheSdkFromAScript.cs

```diff
@@ -16,6 +16,9 @@ namespace Microsoft.DotNet.InstallationScript.Tests
 {
     public class GivenThatIWantToInstallTheSdkFromAScript : TestBase
     {
+        private static string targetBranch = new EnvironmentProvider().GetEnvironmentVariable("SYSTEM_PULLREQUEST_TARGETBRANCH");
+        private readonly bool targetingMaster = (!string.IsNullOrEmpty(targetBranch) && targetBranch.Equals("master", StringComparison.OrdinalIgnoreCase));
+        private static string message = "InstallationScript tests are designed to run only in a CI context and only when the PR targets 'master'.";
```

Comment:
Do the install script tests not run locally?  I would think they should.  Also if the SYSTEM_PULLREQUEST_TARGETBRANCH variable isn't set, I think it would be better to run them anyway.  Otherwise if CI setup changes and the variable isn't set we would silently be getting no coverage.

Reply:
@dsplaisted 
The install script tests can run locally; but they are off by default. As they should be because a CLI developer will not be interested in tests for the install scripts.

Reply:
The CLI developer also shouldn't be making changes to the install scripts.  Do they add significant overhead to test runs?

My main concern is that with the current logic we could end up with the tests never being run and we wouldn't know it.

Reply:
"Do they add significant overhead to test runs?" the tests rely heavily on the network and blob storage - therefore A = normally they are quick to run.